### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by jul3x**

### CODE REVIEW

File: src/Graphics/CarGraphics.cpp [hunk 1]
```diff
@@ -64,6 +64,24 @@ const sf::CircleShape& CarGraphics::getRearWheel() const {
...
+    new_color.a = 80;
+    wheel_front_.setFillColor(new_color);
+
+    new_color = wheel_rear_.getFillColor();
+    new_color.a = 80;
+    wheel_rear_.setFillColor(new_color);
+}
+
+bool CarGraphics::isAlive() const {
+    return car_body_graphics_.getFillColor().a > 250;
```
Comment (danielm1405) [inline src/Graphics/CarGraphics.cpp]: `isAlive()` based on color???

File: test/CMakeLists.txt.in [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.2)
```
Comment (danielm1405) [inline test/CMakeLists.txt.in:1]: `CMakeLists.txt.in`? what does `in` mean?

Comment (jul3x) [reply test/CMakeLists.txt.in:1]: it is input file parsed by CMake and modified depending on what is needed - provided by GTest

File: src/Graphics/CarGraphics.cpp (hunk 1 above)
Comment (jul3x) [reply src/Graphics/CarGraphics.cpp]: todo xD

Comment (jul3x) [reply src/Graphics/CarGraphics.cpp]: done

**Comment by jul3x** [comment]
@danielm1405 merge?

**Comment by danielm1405** [comment]
merge

**Pull request merged**